mne/viz/_3d.py

```diff
@@ -444,10 +480,10 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         mesh.data.cell_data.normals = None
         mlab.pipeline.surface(mesh, color=colors[key], opacity=alphas[key])
 
-    datas = (eeg_loc, hpi_loc, car_loc, ext_loc)
-    colors = ((1., 0., 0.), (0., 1., 0.), (1., 1., 0.), (1., 0.5, 0.))
-    alphas = (1.0, 0.5, 0.5, 0.25)
-    scales = (0.005, 0.015, 0.015, 0.0075)
+    datas = (eeg_loc, eegp_loc, hpi_loc, car_loc, ext_loc)
+    colors = ((1., 0, 0), (0.5, 0, 0), (0., 1, 0), (1., 1, 0), (1, 0.5, 0))
```

Comment:
try:

    colors = ((1., 0, 0), (0., 0.9, 1.), (0., 1, 0), (1., 1, 0), (1, 0.5, 0))

to me it's much nicer :)